mathics/builtin/list/constructing.py

```diff
@@ -371,6 +372,7 @@ class Sow(Builtin):
     </dl>
     """
 
+    summary_text = "send an expression to the innermost Row"
```

Comment:
According to https://reference.wolfram.com/language/ref/Sow.html this is nearest enclosing "Reap" which I think is different.

Reply:
je, dyslexia...